puppeteer_scrape.js

```diff
@@ -0,0 +1,37 @@
+// A script to scrape our TodoMVC app using Puppeteer.
+// To show the scraping browser window as it's happening, set the DEBUG environment variable.
+
+(async () => {
+  const puppeteer = require('puppeteer')
+  const httpServer = require('http-server')
+
+  // Start a local HTTP Server to host our TodoMVC app.
+  const server = httpServer.createServer()
+  const PORT = 8000
+  server.listen(PORT)
+  const URL = `http://localhost:${PORT}`
+
+  // Start a Puppeteer instance for our scraping.
+  const browser = await puppeteer.launch(
+    process.env.DEBUG
```

Comment:
Just an idea. I explained in the `README`: if you set this env variable when you run the script, you can see Puppeteer in action in slow-mo.

I thought it might be neat for folks that may not be super-familiar with Puppeteer or browser automation to see that, but we can easily take it out. I don't know if the intended audience for this example and script are likely to be Puppeteer newbies or pros or what. And totally agreed that this just adds a little bit of distracting complexity to the script.

Reply:
Removed!